SparkPost/2web2ui
# SA-2220 - (UI) Migrate from the `moment-timezone` package to `timezone-support` for listing timezones

[SA-2220](https://sparkpost.atlassian.net/browse/SA-2220)

### What Changed

- Installs `timezone-support` to generate timezone options for the `TimezoneTypeahead` component
- Unfortunately, uninstalling `moment-timezone` was causing a significant bug in the Analytics Report feature - will cut a follow-up to refactor and remove it
- Refactors `TimezoneTypeahead` to allow passed in options as a prop

### How To Test

- Go to `signals/analytics` and verify the timezone typeahead works the network requests made to the metrics endpoints
- Go to `/signals/schedule/<id>` (both the create and edit flows) and verify the timezone typeahead works with network requests

### To Do
